src/Backend/PostponeRuntimeBackend.php

```diff
@@ -79,10 +79,8 @@ public function onEvent(Event $event = null)
     {
         reset($this->messages);
 
-        while (list($key, $message) = each($this->messages)) {
+        foreach ($this->messages as $key => $message) {
             $this->handle($message, $this->dispatcher);
-
-            unset($this->messages[$key]);
```

Comment:
Maybe we should clear this variable after the foreach? To avoid changing behavior